i18n/fr/docusaurus-plugin-content-docs/version-22.04/graph-views/map-web-known-issues.md

```diff
@@ -5,8 +5,10 @@ title: Problèmes connus dans MAP
 
 Cette page répertorie les incidents connus sur lesquels l'équipe Centreon travaille actuellement :
 
+- L'indicateur de statut fonctionne sauf pour les groupes d'hôtes et les groupes de services où l'affichage n'est pas correct.
```

Comment:
C'est pas tout à fait ça. En fait le truc c'est qu'il s'affiche alors qu'un host group ou un service group n'a pas de status propre à lui même. Le status hérité (cad calculé à partir du status qui sont incluses dans le groupe) représenté par le fond est lui correct.
Une meilleure formulation serait:
L'indicateur de statut s'affiche pour les groupes d'hôtes et de services alors que ceux ci n'ont pas de statut propre.
Status box is displayed for host and service groups although they do not have status of their own (?)